lib/ocpp1_6/websocket/websocket.cpp

```diff
@@ -6,18 +6,19 @@
 #include <ocpp1_6/types.hpp>
 #include <ocpp1_6/websocket/websocket.hpp>
 
-namespace ocpp1_6 {
+#include <boost/algorithm/string.hpp>
 
-Websocket::Websocket(std::shared_ptr<ChargePointConfiguration> configuration, int32_t security_profile) {
+using json = nlohmann::json;
 
-    auto uri = configuration->getCentralSystemURI();
+namespace ocpp1_6 {
 
-    this->log_messages = configuration->getLogMessages();
-    if (this->log_messages) {
-        std::string output_file_name = "/tmp/libocpp_messages_" + DateTime().to_rfc3339() + ".txt";
-        this->output_file.open(output_file_name);
-    }
+Websocket::Websocket(std::shared_ptr<ChargePointConfiguration> configuration, int32_t security_profile,
+                     std::shared_ptr<MessageLogging> logging) {
 
+    this->configuration = configuration;
```

Comment:
It is good practice here to initialize member variables with the constructor initializer list like
``Websocket::Websocket(...) : configuration(configuration), logging(logging)``